### PR TITLE
Refactor login alerts to a class for cleanliness

### PR DESCRIPTION
**Pull request by iondune**
http://jira.csc.calpoly.edu/browse/MYS-212

### CODE REVIEW

File: www/js/controllers.js [hunk 1]
```diff
@@ -25,6 +25,48 @@ module.run(function($ionicPlatform) {
...
+
+LoginAlerts.Helper_SuccessAlert = function ($ionicPopup, message) {
+	var alertPopup = $ionicPopup.alert(
+	{
+		title: '<span class="bold success">Success!</span>',
+		template: message
+	});
+};
+
+LoginAlerts.Error_AlreadyLoggedIn = function ($ionicPopup) {
```
Comment (athattayathu) [inline www/js/controllers.js:30]: are you sure we should keep these in the class? seems like these are one off messages that can be placed in the respective controllers. It maybe useful to have convenience methods in the respective controllers and not this class for reuse....

Comment (iondune) [inline www/js/controllers.js:30]: Keep which, the generic Helper_ or the ones specific to the controller?

Comment (athattayathu) [inline www/js/controllers.js:30]: keep the generic ones in the LoginAlerts class and specific ones as helper methods in their controllers if they need to be reused.

**Comment by iondune** [comment]
I moved the generic methods to a new class and put it in utils.js. I'm hesitant to put the login specific alerts into the controller because it is AppCtrl, not a login-specific controller

**Comment by athattayathu** [comment]
makes sense to me

**Pull request merged**